### PR TITLE
Remove lines that cause errors in Anki 2.1

### PR DESCRIPTION
**Pull request by khonkhortisan**
https://github.com/ospalh/anki-addons/issues/117

### CODE REVIEW

**Comment by ospalh** [comment]
Hmm.
This looks like it won’t remove the deck title when you go back to the overview screen.
(Also when you switch off the subdeck function, but that is easy to fix.)

**Comment by ospalh** [comment]
… as you said at #117

**Comment by ospalh** [comment]
Oh, anyway. This looks like a step in the right direction. Thanks.

**Comment by ospalh** [comment]
And maybe i will look into this a bit now, maybe not.
If you have to have this, you’ll still be better off when you do it yourself.

Also, i think the top bits in the file would work great with the new config scheme. See e.g. my [add note id](https://github.com/ospalh/anki-addons/tree/develop/add_note_id). Maybe i will do this. See above.

File: deck_name_in_title/__init__.py [hunk 1]
```diff
@@ -98,8 +98,11 @@ def card_title(self):
 
 
 deck_namer = DeckNamer()
-mw.deckBrowser.show = wrap(mw.deckBrowser.show, deck_namer.deck_browser_title)
-mw.overview.show = wrap(mw.overview.show, deck_namer.overview_title)
-mw.reviewer.show = wrap(mw.reviewer.show, deck_namer.overview_title)
+#mw.deckBrowser.show = wrap(mw.deckBrowser.show, deck_namer.deck_browser_title)
```
Comment (khonkhortisan) [inline deck_name_in_title/__init__.py]: using these has no effect
addHook("showDeckOptions", deck_namer.deck_browser_title)
#addHook("???", deck_namer.overview_title)
addHook("showQuestion", deck_namer.overview_title)

**Pull request merged**